.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,57 +1,131 @@
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
+**Which is the sponsoring or primary SIG associated with this PR?**
+<!--
+Label the sponsoring or primary SIG associated with this PR.
+
+For reference on available SIGs, you can find more details at:
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#labels-that-apply-to-all-repos-for-both-issues-and-prs
+
+Add only ONE sig here via a /sig-primary <name> entry.
 -->
+/sig-primary ???
+
+> DISCUSSION REQUIRED
+>
+> Should we use the above pattern and update the label prow plugin ? https://github.com/kubernetes/test-infra/tree/master/prow/plugins/label. Logic would have to ensure that there is only a single primary sig associated. This metadata can be leveraged by other processes outside of the release notes.
+> 
+> or, if this is ONLY relevant to release notes then, we can add here as a `` ```primary-sig`` code block.
 
 **What type of PR is this?**
-> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
+<!-- 
+Label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. 
+
+For reference on required PR labels, you can find more details at:
+https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
+
+Add only ONE of the following /kind <type> entries.
+
+/kind api-change
+/kind bug
+/kind cleanup
+/kind design
+/kind documentation
+/kind failing-test
+/kind feature
+/kind flake
+-->
+/kind ???
+
+> DISCUSSION REQUIRED
 >
-> /kind api-change
-> /kind bug
-> /kind cleanup
-> /kind design
-> /kind documentation
-> /kind failing-test
-> /kind feature
-> /kind flake
+> Should we include a kind type of `deprecation`? And should we then include a `` ```depracation-notes`` code block to provide specific detailed notes around the depracation.
+> 
+> Should we only allow a single kind to be specified? There are a number of examples where multiple kind values make sense.
 
 **What this PR does / why we need it**:
+<!--
+Provide a description on what this PR does and/or why it is required.
+-->
+Description here ...
 
 **Which issue(s) this PR fixes**:
 <!--
-*Automatically closes linked issue when PR is merged.
-Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
+Add references to issues that are fixed by this PR by using one or more of the following templates:
+
+Fixes #<issue number>
+Fixes <issue url>
+
+NOTE: The linked issue will automatically be closed when the PR is merged.
+
+If the PR is about failing-tests or flakes, please post the related issues/tests in a comment and do not use the Fixes templates above.
+-->
+Fixes #???
+
+**Have you added or run tests for your PR?**
```

Comment:
Can you elaborate on why this is needed? There are lots of PRs that might not need tests :)

Reply:
Perhaps something more generic like "Has consideration been given to a test plan related to this PR?  If appropriate, were tests added/modified to cover this PR?"

I definitely like having a prompt regarding quality assurance.

Reply:
There was an original comment in the header of the PR template:
```
Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
```

I was moving comments around to place them closer to where they had the most context. There was nothing asking the question about tests, so I added a new section. @tpepper - I like your suggestion. It is closer to what I had in my head for this section. @nikhita - if you are ok with Tim's suggestion, I'll update this section with it.

Reply:
> Has consideration been given to a test plan related to this PR? If appropriate, were tests added/modified to cover this PR?

Maybe I'm missing something here, but how would this help us in signaling if tests were added? To elaborate -- I see all the other prompts in the issue templates as "taking some action" depending on the authors' response. For example, applying the release note label or closing an issue...

I feel like the comment in the PR header gives the "fyi, it'd be nicer if you added tests" kind of signal right now, which IMHO is good enough. Wdyt?

Reply:
Should we consider an additional signal around tests for a PR? I assume if so, this would take the form of another label.

My concern would be as to how useful that signal would be. It would be useful if we want to understand how many PRs that should be accompanied by tests are not. But would not be useful for a PR that doesn't require tests and doesn't have a label - how would you differentiate that from a PR that requires tests and is missing them.

If, after more discussion, there is a requirement for a signal around test coverage for PRs, then we should reconsider. For now, I'll move this back to the header comment, but with the updated text suggested by @tpepper.